hardware/bluetooth/updateVendorIds.py

```diff
@@ -0,0 +1,153 @@
+#!/usr/bin/env python3
```

Comment:
It's nice that you wrote a script for downloading the vendor IDs and putting them into the `bluetooth_vendors_ids.ksy` spec, but you don't expect to get this Python script merged into the official KS format gallery repo, do you? I suggest placing it into your own repo instead, and you can add a link to that repo into the `doc` key of `bluetooth_vendors_ids.ksy` spec. Let's keep the https://github.com/kaitai-io/kaitai_struct_formats repo reserved for `.ksy` format specifications.

Reply:
>but you don't expect to get this Python script merged into the official KS format gallery repo, do you?

Of course I do. This script makes no sense without the file it is intended to update. And the file to be updated will rot without this script. Of they both should belong to the same repo.

>I suggest placing it into your own repo instead

It may make sense to put the whole `bluetooth` dir into an own repo and link it as a submodule.